charts/kyverno/templates/reports-controller/flowschema.yaml

```diff
@@ -0,0 +1,92 @@
+{{- if .Values.reportsController.apiPriorityAndFairness }}
+apiVersion: {{ template "kyverno.flowcontrol.apiVersion" . }}
+kind: FlowSchema
+metadata:
+  name: {{ template "kyverno.reports-controller.name" . }}
+  labels:
+    {{- include "kyverno.reports-controller.labels" . | nindent 4 }}
+spec:
+  priorityLevelConfiguration:
+    name: {{ template "kyverno.reports-controller.name" . }}
+  rules:
+  - resourceRules:
+    - apiGroups:
+        - '*'
+      clusterScope: true
+      resources:
+        - '*'
+      verbs:
+        - get
+        - list
+        - watch
+    - apiGroups:
+        - kyverno.io
+      clusterScope: true
```

Comment:
should this entry contain only cluster scope resources ?

Reply:
yes

Reply:
`admissionreports` for example is not a cluster scope resourrce 🤔